examples/simple-nft-minter/index.mjs

```diff
@@ -0,0 +1,63 @@
+import { Reach, test } from '@reach-sh/stdlib';
+const stdlib = new Reach(process.env, { REACH_NO_WARN: 'Y' });
+
+const startingBalance = stdlib.parseCurrency(100);
+
+const [minter, receiver] = await stdlib.newTestAccounts(2, startingBalance);
+
+const gasLimit = 500000;
+if (stdlib.connector == 'ETH') {minter.setGasLimit(gasLimit)};
```

Comment:
This should also be applied to `CFX`. You could just do `minter.setGasLimit(gasLimit)` without the `if` condition because it is a no-op when using `ALGO`.